Salesforce Marketing Cloud (ExactTarget) documentation

mohan-h opened the pull request:
Initial commit

juliette-massardier committed:
Update license-usage.md

Hello, as discussed with the Adobe Journey Optimizer product team, we should add the AJO licenses listed here and the link to the AJO product description. Thanks! Juliette (AJO doc team)

JCON3DEV committed:
Merge pull request #80 from juliette-massardier/patch-1

Update AJO license-usage.md

mohan-h committed:
Merge branch 'AdobeDocs:main' into main

mohan-h committed:
Merge branch 'AdobeDocs:main' into main

mohan-h committed:
Merge branch 'AdobeDocs:main' into main

mohan-h committed:
Merge branch 'AdobeDocs:main' into main

mohan-h committed:
Salesforce Marketing Cloud (ExactTarget) documentation

Initial commit

mohan-h committed:
Added errors and troubleshooting section

vgiurgiu commented on help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md:
```diff
@@ -0,0 +1,192 @@
+
+[[!DNL Salesforce Marketing Cloud (ExactTarget)]](https://www.salesforce.com/products/marketing-cloud/overview/) is a digital marketing suite formerly known as ExactTarget that allows you to build and customize journeys for visitors and customers to personalize their experience.
+
+This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [Salesforce Update Contacts REST API](https://developer.salesforce.com/docs/marketing/marketing-cloud/guide/updateContacts.html), which allows you to add contacts / update contact data for your business needs after activating them within a new Salesforce segment.
```
Maybe we should highlight this paragraph in an IMPORTANT note. 

The reason is that we already have a Salesforce Marketing Cloud destination in the UI (https://experienceleague.adobe.com/docs/experience-platform/destinations/catalog/email-marketing/salesforce-marketing-cloud.html?lang=en). 

We need to clearly indicate the difference between the old and the new destination, namely that the old one exports files and this one is an API-based streaming connection.

mohan-h replied:
Makes sense have added the below instead of highlighting your recommendation which is more nuanced.
```
Kindly note a different [Salesforce Marketing Cloud connection](https://experienceleague.adobe.com/docs/experience-platform/destinations/catalog/email-marketing/salesforce-marketing-cloud.html?lang=en) exists within Email marketing. However, its operation is to support file exports to a specified storage location, whereas this is an API-based streaming connection.
```

vgiurgiu commented on help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md:
```diff
@@ -0,0 +1,192 @@
+
+## Prerequisites {#prerequisites}
+
+Before activating data to the Salesforce Marketing Cloud (ExactTarget) destination, you must have a [schema](https://experienceleague.adobe.com/docs/experience-platform/xdm/schema/composition.html), a [dataset](https://experienceleague.adobe.com/docs/platform-learn/tutorials/data-ingestion/create-datasets-and-ingest-data.html?lang=en), and [segments](https://experienceleague.adobe.com/docs/platfo ...
```
With regard to the format of the prerequisites section, see my comment in the PR for the other other Salesforce CRM connection: 

https://github.com/AdobeDocs/experience-platform.en/pull/60#discussion_r921507818

vgiurgiu commented on help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md:
```diff
@@ -0,0 +1,192 @@
+1. In the select target field window, select the target field and choose the **[!UICONTROL Select identity namespace]** category and add the mappings desired.
+![Target mapping](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/target-mapping.png)
+
+1. To map any custom attributes select target field window, select the target field and choose the **[!UICONTROL Select attributes]** > **Email Demographics** category, Next provide the desired target attribute name and add the mappings desired.
```
Minor wording suggestion:

To map any custom attributes, select the target field window, select the target field and choose the **[!UICONTROL Select attributes]** > **Email Demographics** category. Next, provide the desired target attribute name and add the mappings desired.

vgiurgiu commented on help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md:
```diff
@@ -0,0 +1,192 @@
+
+### Schedule segment export and example {#schedule-segment-export-example}
+
+* When performing the [Schedule segment export](../../ui/activate/activate-segment-streaming-destinations.html?lang=en#scheduling) step you will need to manually map Platform segments to the custom attribute in Salesforce.
```
Same comment as for the other Salesforce destination that you are working on: https://github.com/AdobeDocs/experience-platform.en/pull/60/files#r921524882

vgiurgiu commented on help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md:
```diff
@@ -0,0 +1,192 @@
+### Schedule segment export and example {#schedule-segment-export-example}
+
+* When performing the [Schedule segment export](../../ui/activate/activate-segment-streaming-destinations.html?lang=en#scheduling) step you will need to manually map Platform segments to the custom attribute in Salesforce.
+* To do this, select each segment, then enter the corresponding custom attribute from Salesforce in the **[!UICONTROL Mapping ID]** field.
```
Same comment about bullet points as here: https://github.com/AdobeDocs/experience-platform.en/pull/60/files#r921525736

vgiurgiu commented on help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md:
```diff
@@ -0,0 +1,192 @@
+
+All [!DNL Adobe Experience Platform] destinations are compliant with data usage policies when handling your data. For detailed information on how [!DNL Adobe Experience Platform] enforces data governance, see the [Data Governance overview](/help/data-governance/home.md).
+
+## Errors and troubleshooting {#errors-and-troubleshooting}
```
Same comment about this section as here: 
https://github.com/AdobeDocs/experience-platform.en/pull/60/files#r921531186

vgiurgiu reviewed (commented):
Some comments, most are similar to the comments on the other Salesforce destination that you are currently working on (https://github.com/AdobeDocs/experience-platform.en/pull/60/files)

mohan-h committed:
Updates against review

Updates against review https://github.com/AdobeDocs/experience-platform.en/pull/88#pullrequestreview-1039473792

vgiurgiu commented on help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md:
```diff
@@ -0,0 +1,213 @@
+
+> [!IMPORTANT]
+> 
+> Kindly note a different [Salesforce Marketing Cloud connection](https://experienceleague.adobe.com/docs/experience-platform/destinations/catalog/email-marketing/salesforce-marketing-cloud.html?lang=en) exists within Email marketing. However, its operation is to support file exports to a specified storage location, whereas this is an API-based streaming connection.
```
Let's rephrase a bit. No need to be excessively polite here: 

Note the difference between this connection and the other [Salesforce Marketing Cloud connection](https://experienceleague.adobe.com/docs/experience-platform/destinations/catalog/email-marketing/salesforce-marketing-cloud.html?lang=en) that exists within the Email marketing catalog section. The other Salesforce Marketing Cloud connection allows you to export files to a specified storage location, whereas this is an API-based streaming connection.

vgiurgiu commented on help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md:
```diff
@@ -0,0 +1,213 @@
+
+To help you better understand how and when you should use the Salesforce Marketing Cloud (ExactTarget) destination, here is a sample use case that Adobe Experience Platform customers can solve by using this destination.
+
+### Send emails to contacts for marketing campaigns
```
I didn't catch this earlier, but let's please add an anchor to all headings, down to H3s (everything starting with `###`). This also applies to line 35 and all other places where we have H3 headings.

vgiurgiu commented on help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md:
```diff
@@ -0,0 +1,213 @@
+
+### Unknown errors encountered while pushing events to Salesforce Marketing Cloud
+
+When checking a dataflow run, if you obtain the below error message; Check that the Mapping ID you provided against your Platform segment is valid and exists within Salesforce Marketing Cloud.
```
Thanks for adding the heading! Minor comment that the phrasing in the paragraph can still be improved as suggested here: 

When checking a dataflow run, if you see the error message below, verify that the **Mapping ID** that you provided in [!DNL Salesforce CRM] for your Platform segment is valid and exists within [!DNL Salesforce CRM].

vgiurgiu reviewed (commented):
Almost there! Just a few more minor comments.

mohan-h committed:
Updates against review

Updates against review https://github.com/AdobeDocs/experience-platform.en/pull/88#pullrequestreview-1040422174

mohan-h committed:
Merge branch 'AdobeDocs:main' into main

mohan-h committed:
Merge branch 'AdobeDocs:main' into salesforce-mktg-cloud-exact-target-dest

vgiurgiu commented on help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md:
```diff
@@ -0,0 +1,213 @@
+description: The Salesforce Marketing Cloud (ExactTarget) destination allows you to export your account data and activate it within Salesforce Marketing Cloud (ExactTarget) for your business needs.attribute set
+---
+
+# [!DNL Salesforce Marketing Cloud (ExactTarget)] connection
```
Let's please update to [!DNL (API) Salesforce Marketing Cloud]. We can mention ExactTarget in the description.

vgiurgiu commented on help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md:
```diff
@@ -0,0 +1,213 @@
+
+## Overview {#overview}
+
+[[!DNL Salesforce Marketing Cloud (ExactTarget)]](https://www.salesforce.com/products/marketing-cloud/overview/) is a digital marketing suite formerly known as ExactTarget that allows you to build and customize journeys for visitors and customers to personalize their experience.
```
[[!DNL Salesforce Marketing Cloud]](https://www.salesforce.com/products/marketing-cloud/overview/) (formerly known as ExactTarget) is [...]

vgiurgiu commented on help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md:
```diff
@@ -0,0 +1,213 @@
+---
+keywords: email;Email;e-mail;email destinations;salesforce;salesforce destination
+title: Salesforce Marketing Cloud (ExactTarget) connection
```
Let's please update to (API) Salesforce Marketing Cloud. We can mention ExactTarget in the description.

vgiurgiu commented on help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md:
```diff
@@ -0,0 +1,213 @@
+---
+keywords: email;Email;e-mail;email destinations;salesforce;salesforce destination
+title: Salesforce Marketing Cloud (ExactTarget) connection
+description: The Salesforce Marketing Cloud (ExactTarget) destination allows you to export your account data and activate it within Salesforce Marketing Cloud (ExactTarget) for your business needs.attribute set
```
The Salesforce Marketing Cloud (formerly known as ExactTarget) destination allows you to [...]

vgiurgiu reviewed (commented):
Added some comments about naming the destination

mohan-h committed:
Changes to accomodate card name change

vgiurgiu commented on help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md:
```diff
@@ -0,0 +1,213 @@
+
+> [!IMPORTANT]
+>
+> * The value used for the Mapping ID should exactly match the name of the custom attribute created within Salesforce under the "Email Demographics" attribute-set.
```
Minor comment: no need for the asterisk here in the note.

mohan-h committed:
Merge branch 'AdobeDocs:main' into main

mohan-h committed:
Merge branch 'main' into salesforce-mktg-cloud-exact-target-dest

mohan-h committed:
Updates against review

Updates against review https://github.com/AdobeDocs/experience-platform.en/pull/88#discussion_r926735077

mohan-h committed:
Merge branch 'AdobeDocs:main' into salesforce-mktg-cloud-exact-target-dest

mohan-h committed:
Added limits section

vgiurgiu commented on help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md:
```diff
@@ -0,0 +1,222 @@
+
+### Limits {#limits}
+
+* Dynamics 365 imposes certain [rate limits](https://developer.salesforce.com/docs/marketing/marketing-cloud/guide/rate-limiting.html).
```
Looks like a minor error slipped in. Can you please update Dynamics 365 to Salesforce?

vgiurgiu commented on help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md:
```diff
@@ -0,0 +1,222 @@
+
+* Dynamics 365 imposes certain [rate limits](https://developer.salesforce.com/docs/marketing/marketing-cloud/guide/rate-limiting.html).
+* Refer to the [rate limits errors](https://developer.salesforce.com/docs/marketing/marketing-cloud/guide/rate-limiting-errors.html) page to check any errors that you might encounter.
+* Refer to the [Marketing Cloud Engagement Pricing](https://www.salesforce.com/editions-pricing/marketing-cloud/email/) page to *Download the Full Edition Comparison Chart* as a pdf which details the limits imposed by your plan.
```
just to narrow things down, since Adobe's Experience Cloud was previously called Marketing Cloud and to avoid any confusion, please add Salesforce to the link, like: 

Refer to the [Salesforce Marketing Cloud Engagement Pricing]

vgiurgiu replied:
just to narrow things down, since Adobe's Experience Cloud was previously called Marketing Cloud and to avoid any confusion, please add Salesforce to the link, like: 

Refer to the [Salesforce Marketing Cloud Engagement Pricing]

vgiurgiu reviewed (commented):
Minor comments. Thanks for adding the section.

mohan-h committed:
Updates post review

vgiurgiu commented:
The documentation for the new Salesforce Marketing Cloud destination is live at: https://experienceleague.adobe.com/docs/experience-platform/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.html?lang=en
